### ISSUE TITLE
Regarding unuse login feature

### ISSUE THREAD

**Issue by eugenepeh** [CS2103AUG2017-T12-B3/main#86]
@aggarwalRuchir ,

Since it was an originally working feature, you might want to add the Screenshot of the original state, a short write up in developer guide and probably list out what else left to be done for that feature.

Also, you can add that developer guide portion in your portfolio afterward.

**Comment by aggarwalRuchir** [comment]
@eugenepeh I will finish by today. However, I want to ask whether there will be a penalty for that? Because I did put the code as unused code and put the information about it in the Project Portfolio already but not as a developer guide, rather as a proposed enhancement.

**Comment by eugenepeh** [comment]
Yes, it is too late to make any changes to submitted work. You can still do the work on a PR and <b> seek our approval </b> before merging it into the master branch.

The rationale is that if you put it as proposed enhancement, the grader may not know that its an implemented working feature, so I would suggest that you put it under developer guide.

**Comment by eugenepeh** [comment]
Sorry, my original comment forgot to take your original deadline into account. You can hold on for now, while I clarify further for you.

**Comment by aggarwalRuchir** [comment]
@eugenepeh sure. I can still work on a PR. If its required, I will make it otherwise the current version is fine.

**Comment by eugenepeh** [comment]
@aggarwalRuchir , I have clarified, you can just send an email to Prof. Damith and Mr. Akshay (The head TA) and explain to them in short details why it is unuse, if you would like to claim it as an feature.

**Comment by aggarwalRuchir** [comment]
@eugenepeh so I don't need to make a PR and add it to the Developer Guide?

**Comment by eugenepeh** [comment]
@aggarwalRuchir yup, just briefly explain to them will do. They will take note of it when grading your portion.

**Comment by aggarwalRuchir** [comment]
@eugenepeh yeah I sent them an email of what I did and what is left. I think that would be good enough
